src/Services/ThrottleRequestsService.php

```diff
@@ -9,12 +9,12 @@ class ThrottleRequestsService {
     use AuthenticatesUsers;
 
     public $maxAttempts = 5; // change to the max attempt you want.
-    public $delayMinutes = 1;
+    public $decayMinutes = 1;
 
     public function __construct(int $maxAttempts = 5, int $delayMinutes = 1)
     {
         $this->maxAttempts = $maxAttempts;
-        $this->delayMinutes = $maxAttempts;
+        $this->decayMinutes = $delayMinutes;
```

Comment:
```suggestion
        $this->delayMinutes = $delayMinutes;
```

Reply:
@ikechukwukalu 

Hello, I noticed that you reverted  decayMinutes  back to  delayMinutes . However, it appears that setting  delayMinutes  may not be effective due to the references to Laravel's  AuthenticatesUsers/ThrottlesLogins  in  ThrottleRequestsService  and the method calls within the trait. The  incrementAttempts  method, which is invoked, calls the  incrementLoginAttempts  method from the trait. Inside the  incrementLoginAttempts  method, it checks for the existence of the  decayMinutes  property instead of  delayMinutes . As a result, the current configuration with  delayMinutes  seems to have no impact ？